packages/mds-transaction-service/@types/index.ts

```diff
@@ -28,13 +28,52 @@ export interface PaginationLinks {
 // one example -- many others are possible
 export interface TripReceiptDetailsDomainModel {
   trip_id: UUID
+  /**
+   * Should be populated with the timestamp of the starting event
+   * (trip_enter_jurisdiction | enter_jurisdiction | trip_start) for the trip.
+   * If there is none, then this defaults to the ending event timestamp.
+   */
   start_timestamp: Timestamp
+  /**
+   * Should be populated with the timestamp of the terminus event (trip_leave_jurisdiction | trip_end) for the trip.
+   * If there is none, then this defaults to the starting event timestamp.
+   */
   end_timestamp: Timestamp
-  vehicle_type: VEHICLE_TYPE
```

Comment:
Why are we removing vehicle type. Is it underivable? What does that mean here?

Reply:
Not entirely underivable, but it's a superfluous artifact from the original spec. This would just be the vehicle_type of the vehicle that took the trip. Because the device_id is here already, this could just be looked up if the information is needed.